File: c/5-Arrays/workout/aw-1.c
#include <stdio.h>
int main()
{   int a;
    if(scanf("%d\n",&a)!=1||a<=0)
    {
        printf("Invalid input");
        return 0;
    }
    int arr[a];
    for(int i =0;i<a;i++)
    {
        scanf("%d ",&arr[i]);
    }
    {
        for(int i =0;i<a;i++)
        {
            int isduplicate =0;
            for(int j=0;j<i;j++)
            {
                if(arr[i]==arr[j])
                {
                    isduplicate =1;
                    break;
                }
            }
             if(!isduplicate)
                {
                    printf("%d ",arr[i]);
                }
        }
    }
}

File: java/4-Arrays/workout/aw-1.java
import java.util.Scanner;
public class A{
public static void main(String[] args){
    

    Scanner sc = new Scanner(System.in);
    int a =sc.nextInt();
    
    if(a<0)
    {
        System.out.println("Invalid input");
    }
    else{
    int arr[] = new int[a];
    for(int i =0;i<a;i++)
    {
        arr[i] =sc.nextInt();
        System.out.print(arr[i]+" ");
    }
    }
}  
}